draft-ietf-quic-recovery.md

```diff
@@ -1065,6 +1004,16 @@ kLossReductionFactor:
 : Reduction in congestion window when a new loss event is detected.
   The RECOMMENDED value is 0.5.
 
+kPersistentCongestion:
+: Number of consecutive PTOs after which network is considered to be
+  experiencing persistent congestion.  The rationale for this threshold is to
+  enable a sender to use initial PTOs for aggressive probing, similar to Tail
+  Loss Probe (TLP) in TCP {{TLP}} {{RACK}}.  Once the number of consecutive PTOs
+  reaches this threshold - that is, persistent congestion is established - the
+  sender responds by collapsing its congestion window, similar to a
+  Retransmission Timeout (RTO) in TCP {{RFC5681}}.  The RECOMMENDED value is 3,
```

Comment:
The pseudocode below decreases CWND to kMinimumWindow, which is 2 packets.  I think sticking with that makes sense, at least for now.  It also requires less text to justify.

Reply:
The recommendation is 3 for kPersistentCongestion, not for the value of cwnd. I'll change the text a bit to clarify.